src/Opdex.Platform.Application/PlatformApplicationServiceCollectionExtensions.cs

```diff
@@ -684,4 +687,10 @@ private static IServiceCollection AddAssemblers(this IServiceCollection services
 
         return services;
     }
+
+    private static IServiceCollection AddCacheServices(this IServiceCollection services)
```

Comment:
Curious about naming, "Cache" doesn't seem very fitting here right?

Reply:
That's going to require a redeployment or restart of the API when we deploy a new FN version. 

Thinking now that the list is retrieved from the FN, we add a `Trusted` DB column to the `token_wrapped` table and set it once during token creation. Then we don't need to cache anything and the list would only be retrieved when adding a new token.